apps/trading/components/fees-container/fees-container.tsx

```diff
@@ -461,29 +483,37 @@ const VolumeTiers = ({
     <div>
       <Table>
         <THead>
-          <tr>
-            <Th>{t('Tier')}</Th>
-            <Th>{t('Discount')}</Th>
-            <Th>{t('Min. trading volume')}</Th>
-            <Th>
+          <Tr>
+            <Th testId="tier-header">{t('Tier')}</Th>
+            <Th testId="discount-header">{t('Discount')}</Th>
+            <Th testId="min-volume-header">{t('Min. trading volume')}</Th>
+            <Th testId="my-volume-header">
               {t('myVolume', 'My volume (last {{count}} epochs)', {
                 count: windowLength,
               })}
             </Th>
-            <Th />
-          </tr>
+            <Th testId="actions-header" />
+          </Tr>
         </THead>
         <tbody>
           {Array.from(tiers).map((tier, i) => {
             const isUserTier = tierIndex === i;
 
             return (
-              <Tr key={i}>
-                <Td>{i + 1}</Td>
-                <Td>{formatPercentage(Number(tier.volumeDiscountFactor))}%</Td>
-                <Td>{formatNumber(tier.minimumRunningNotionalTakerVolume)}</Td>
-                <Td>{isUserTier ? formatNumber(lastEpochVolume) : ''}</Td>
-                <Td>{isUserTier ? <YourTier /> : null}</Td>
+              <Tr key={i} testId={`tier-row-${i}`}>
```

Comment:
(non blocking): I find it a tad overkill here to add a testid for everything in the table, including the index. Seeing as you'll presumably be using the index in the test code anyway would it be cleaner to just grab all the rows and reference them by index from the resulting array?